ajax/libs/zingchart-angularjs/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "zingchart-angularjs",
+  "filename": "zingchart-angularjs.min.js",
+  "version": "1.2.0",
+  "description": "A ZingChart AngularJS directive",
+  "keywords": [
```

Comment:
@NiniHsu You can use the keywords here https://github.com/zingchart/ZingChart-AngularJS/blob/master/bower.json#L6